mongo/database_internal_test.go

```diff
@@ -53,3 +53,16 @@ func TestDatabase_RunCommand(t *testing.T) {
 	require.Equal(t, ok.Value().Type(), bson.TypeDouble)
 	require.Equal(t, ok.Value().Double(), 1.0)
 }
+
+func TestDatabase_Drop(t *testing.T) {
+	t.Parallel()
+	name := "test"
```

Comment:
Yeah. It will be good to use not used database name. Buf if I use ```testutil.DBName``` I get the error in collection test because I dropped shared database. So maybe we need for this test some random name for the database with a timestamp.

Reply:
Ah, you're right! I had conflated the behavior of `testutil.DBName` with `testutil.ColName`. Instead of having the random name, just make the name `TestDatabase_Drop`, since we shouldn't have any databases with that name. I'd rather not add more methods to the testutil package because we'll eventually need to get rid of it. Sorry for the confusion there.

Reply:
Thank you for your clarification.